### PR TITLE
docs: add assistants api reference

### PR DESCRIPTION
**Pull request by hieu-jan**
- [x] Solved issue #688 
- [x] Adding the requestBody and response for each assistant endpoint API
- [x] Run auto format for all .yaml files to make the code clean
- [x] Fixing some minor bugs in the API Reference page

### CODE REVIEW

File: docs/openapi/specs/assistants.yaml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                type: string
+            out:
+              type: array
+              items:
+                type: string
+          # If there are specific event types, they can be detailed here
+        metadata:
+          type: object
+          description: "Metadata associated with the assistant."
+      required:
```
Comment (0xSage) [inline docs/openapi/specs/assistants.yaml:58]: afaik none of the fields in assts are required

**Review by 0xSage — COMMENTED** [review]
@hieu-jan a more accurate reference for assistants is here: http://localhost:3001/specs/assistants/#assistantjson

The github description is a bit outdated.

Pls fix then we're ready to merge! Thank you

**Comment by hieu-jan** [comment]
I have updated the assistant model. Currently fixing the request and response content of each API endpoint.

**Comment by hieu-jan** [comment]
Hi @0xSage, I made some new change to the current version including
- [x] Adding requestBody and response for each assistant endpoint API
- [x] Run auto format for all .yaml files to make the code clean
- [x] Fixing some minor bugs of links, text in Models, Messages

**Review by 0xSage — CHANGES_REQUESTED** [review]
Hey, things look a bit better, but I think I mentioned in a previous PR that non of these fields are required: 

image

**Comment by hieu-jan** [comment]
Hi @0xSage, sorry for confusing you, I guess this image is from the old deployment version. In the current latest version, I already fixed this one too. https://f0d8c9d9.jan-qn8.pages.dev/api-reference

**Pull request merged**